lib/modules/dosomething/dosomething_mbp/dosomething_mbp.install

```diff
@@ -188,8 +199,8 @@ function dosomething_mbp_install_productions() {
   $productions[4]['queues'][] = 'transactionalQueue';
   $productions[4]['queues'][] = 'loggingQueue';
   $productions[4]['queues'][] = 'activityStatsQueue';
-  $productions[4]['queues'][] = 'userCampaignActivityQueue';
```

Comment:
Nope.

The `loggingQueue`: https://github.com/DoSomething/mbc-logging/blob/master/app.js#L30 supports the activity stream for logging and `activityStatsQueue` feeds the dashboard via `mbp-externalApplications-dashboard`: https://github.com/DoSomething/mbp-externalApplications-dashboard. 

I created `userCampaignActivityQueue` quite awhile ago for a different purpose that's no longer needed. This is an effort to clean things up a bit while adding other, unrelated functionality for image processing.